### DISCUSSION TITLE
What`s the mean of "np"?

### DISCUSSION THREAD

**Discussion by suucss** [3b1b/manim#1540] [General]
It is custom to use "np.array[ x, y ]", but I am confused by the name it called. Is it represent the meaning of "[n]ow [p]lace (place of now)"?

**Comment by otakucode** [comment]
It's short for "numpy" which is the name of the Python module the data type
comes from.

**Reply by suucss** [reply]
Useful anwser!

**Comment by TonyCrane** [comment]
If you take a look at `manimlib/__init__.py` you will find `import numpy as np`.

**Reply by suucss** [reply]
厉害！
